src/libraries/evio/HDEVIO.cc

```diff
@@ -103,7 +103,8 @@ HDEVIO::HDEVIO(string fname, bool read_map_file, int verbose):filename(fname),VE
 HDEVIO::~HDEVIO()
 {
 #ifdef USE_ASYNC_FILEBUF
-	delete ifs.std::ios::rdbuf();
+    _DBG_ << "Call HDEVIO destructor!" << ifs.std::ios::rdbuf() << std::endl;
+//    delete ifs.std::ios::rdbuf();   /// @davidl
```

Comment:
@faustus123 I remove this delete clause since it will cause "double free or corruption" error. But by removing it, I am not sure whether it will cause memory leakage.

Reply:
Any memory leakage should be negligible since this will only happen once for every file processed. 

We seem to be having this double free error in a lot of places all of a sudden. EICrecon also has an issue. Are you seeing it with the latest JANA v2.0.9? There was some speculation that was a recent change that could be causing it.

Regardless, this should be fine. I will go ahead and merge it. I have not been able to test (too many fires to put put). I don't want to wait too long to merge it into main since it seems more likely we'll run into conflicts later.

Reply:
I just compiled JANA2 on gluon200 to latest version, and this double free error is still there.

`jana -v`  does not have this problem.